### PR TITLE
Automated PR for FTL

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/FTL.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      <p>The FreeType Project is distributed in several archive packages; some of them may contain, in addition to
+         the FreeType font engine, various tools and contributions which rely on, or relate to, the FreeType
+         Project.</p>
+      <p>This license applies to all files found in such packages, and which do not fall under their own explicit
+         license. The license affects thus the FreeType font engine, the test programs, documentation and
+         makefiles, at the very least.</p>
+      <p>This license was inspired by the BSD, Artistic, and IJG (Independent JPEG Group) licenses, which all
+         encourage inclusion and use of free software in commercial and freeware products alike. As a
+         consequence, its main points are that:</p>
+      <p>o We don't promise that this software works. However, we will be interested in any kind of bug reports.
```
Comment (sam-ellis) [inline src/FTL.xml]: These 'o' need converting to <list> tags.

Comment (jlovejoy) [inline src/FTL.xml]: not sure what you mean?

**Comment by bradleeedmondson** [comment]
oh those look like they should be wrapped in bullet tags

File: src/FTL.xml (hunk 1 above)
Comment (sam-ellis) [inline src/FTL.xml]: The 'o' characters are actually bullets and so these are the related text ought to be converted into list tags. Pull request submitted.

File: src/FTL.xml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+             authors, grants you the right to use, distribute, and modify it. Therefore, by using,
+             distributing, or modifying the FreeType Project, you indicate that you understand and accept
+             all the terms of this license.</p>
+        </li>
+        <li>
+          <b>4.</b>
+          <p>Contacts</p>
+        </li>
+      </list>
+      <p>There are two mailing lists related to FreeType:</p>
```
Comment (sam-ellis) [inline src/FTL.xml]: This text should be within the Contacts bullet. Pull request submitted.

Comment (jlovejoy) [inline src/FTL.xml]: was this fixed? I cna't tell...

Comment (bradleeedmondson) [inline src/FTL.xml]: no, sam had submitted a PR against kris' repo, but neither kris nor I had merged it. 

I merged it just now, so the current SPDX PR reflects the fixed version (with mailing list paragraphs moved up a bit and into the &lt;li&gt; for Contacts.

**Comment by bradleeedmondson** [comment]
There were 3 pull requests from Sam against Kris' repo that I had missed
back in November, meaning the 3 files he fixed wouldn't have showed those
fixes in the PR against the SPDX repo. Those have now been merged and the
respective PRs against the SPDX repo are ready for final review.

Thanks,
Brad

--
Brad Edmondson, *Esq.*

**Pull request merged**